README.md

```diff
@@ -1,5 +1,7 @@
 # Electron [![Build Status](https://travis-ci.org/atom/electron.svg?branch=master)](https://travis-ci.org/atom/electron)
```

Comment:
Maybe nuke this heading and put the build badge under the logo like https://github.com/atom/atom#readme ?